README.md

```diff
@@ -1489,6 +1489,7 @@ Where to discover new Go libraries.
 
 ## Conferences
 
+* [Capital Go](http://www.capitalgolang.com) - Washington DC, USA
```

Comment:
Minor, but shouldn't this be "Washington, D.C., USA"?

That's what it says at http://www.capitalgolang.com, and <https://en.wikipedia.org/wiki/Washington,_D.C.>.